client/blocks/app-banner/style.scss

```diff
@@ -0,0 +1,64 @@
+.card.app-banner {
+	position: fixed;
+	bottom: 0;
+	width: 100%;
+	background-color: $blue-light;
+	color: $blue-dark;
+	/* needed for banner to appear in front of notifications panel */
+	z-index: 180;
```

Comment:
Don't forget to add this to _z-index.scss. https://chrome.google.com/webstore/detail/z-context/jigamimbjojkdgnlldajknogfgncplbh can help with finding the parent context.